design/FY2019/Interzone Air Wall NFP and Design.md

```diff
@@ -0,0 +1,192 @@
+more complicated.
+
+### Radiant Exchange ###
+For surface-to-surface radiant exchange, all surfaces in the zones connected by airwalls will be combined into
+a superzone for the radiant exchange calculations. Radiant gains from lights, people, HVAC equipment, etc. will 
+be distributed to all surfaces in the superzone.
+
+#### Pitfalls ####
+ - Radiant exchange calculations become very slow for zones with a large number of surfaces. This approach
+ will exaggerate that problem.
+ - Within the superzone it is likely that many surfaces will not have a direct view to all of the surfaces
+ in the combined zone. Presently there is no algorithm to exclude certain views. View factors could be determined
+ external to EnergyPlus and imported using `ZoneProperty:UserViewFactors:bySurfaceName`.
+ - Input processing will need to allow view factors (for radiant HVAC equipment) to cross zone boundaries.
+ - Re-simulation for radiant HVAC equipment will involve all zones in the superzone, causing extra computation time.
+ - Reporting for zone-level radiant gains may be confusing. For example, radiant gains which have a source in Zone A
+ may be distributed to other zones as well as Zone A. Even though the source of the gain is Zone A, the other zones
+ should be reporting their share of the distributed gain.
+ 
+### Solar Gains and Daylighting ###
+For solar distribution and daylighting calculations, the zones connected by airwalls will be combined into a superzone.
+
+#### Pitfalls ####
+ - For `FullInteriorAndExterior` solar distribution options, the convex zone limitation will likely be tripped
+ by the combined zones. The ultimate goal would be to crack this nut once and for all for any situation (airwalls or not).
+ But the fallback position is to figure out a way to trap excess solar that has made it past an interior obstruction and
+ remove it - if possible.
+ - Reporting for solar gains may be confusing. For example, window heat gain calculations will need to
+ account for transmitted solar which passes through to another zone (and the reverse).
+
+### Air Exchange ###
+For simple air exchange (not AirflowNetwork) the user will be expected to input `ZoneMixing` or `ZoneCrossMixing`
+objects the same as always. For simulations using AirflowNetwork, it gets a little more complicated. For smaller openings,
+such as open doorways, the current `AirflowNetwork:MultiZone:Component:DetailedOpening`, 
+`AirflowNetwork:MultiZone:Component:SimpleOpening` and `AirflowNetwork:MultiZone:Component:HorizontalOpening`
+objects should be adequate. For very large openings (horizontal and vertical) another approach will be required. One
+possibility is to use a superzone for the AirflowNetwork calculations. Or perhaps a new opening type could be developed
+to approximate the connection.
+
+#### Pitfalls ####
+ - It's unclear what the values should be for the simple mixing models across an open boundary.
```

Comment:
Might consider an input value for Air Changes Per Hour for the opening.  I usually recommend 1 ACH based on the volume of the smaller zone as a basis for determining input for an airflow rate in zone mixing objects.  This might be automated to make it easier for the user with an ACH input to scale, such as higher for horizontal orientation.  I guess this would lead to an air wall construction object that gets referenced.  There might be other inputs that might go into a Construction:AirWall object.

Reply:
For what it's worth this is what OpenStudio does.  The way you tag an AirWall in OpenStudio is to assign a construction having a single layer of [AirWallMaterial](https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/model/AirWallMaterial.hpp).  We added this as a material type with the intention to hang these types of properties onto it (e.g. ACH, transparent, etc).  That way you can also assign different properties to different airwalls in your model.

Reply:
Construction:AirWall would be just as good, I can't remember why we used a material instead of a construction to indicate airwalls.

Reply:
Sure, I'll consider a construction and/or material type that could be used to control properties of the boundary. Perhaps this should replace Surface Type = AirWall as the trigger? Then the surfaces would still be Wall, Roof, Floor, etc. and the Construction with Material:AirWall would be the trigger to treat this surface differently.

Reply:
I'm not a fan of automating the "simple mixing models" or attaching it to the material. Construction, maybe, but shouldn't the air transfer be based on physics?